src/testUtils/java/org/broadinstitute/hellbender/testutils/FastaTestUtils.java

```diff
@@ -0,0 +1,101 @@
+package org.broadinstitute.hellbender.testutils;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.samtools.reference.FastaSequenceIndex;
+import htsjdk.samtools.reference.IndexedFastaSequenceFile;
+import htsjdk.samtools.reference.ReferenceSequence;
+import htsjdk.samtools.reference.ReferenceSequenceFileFactory;
+import htsjdk.samtools.util.SequenceUtil;
+import org.broadinstitute.hellbender.engine.ReadsDataSource;
+import org.broadinstitute.hellbender.utils.fasta.CachingIndexedFastaSequenceFile;
+import org.broadinstitute.hellbender.utils.reference.FastaReferenceWriter;
+import org.testng.Assert;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Map;
+
+public class FastaTestUtils {
+
+    public static void assertOutput(final Path path, final boolean mustHaveIndex, final boolean mustHaveDictionary,
```

Comment:
Add docs for these methods (and give `assertOutput()` a more descriptive name)

Reply:
I moved most of these back to the unit test they originally came from.  I had intended to use them in my tests but they ended up being to brittle to adapt.  I documented what was left.